src/popup/components/AccountCard.vue

```diff
@@ -8,67 +8,67 @@
   >
     <div class="top">
       <AccountInfo
-        :account-idx="accountIdx"
+        :account="account"
         :color="color"
       />
     </div>
 
     <div class="middle">
-      <BalanceInfo
-        :account-idx="accountIdx"
-      />
+      <BalanceInfo :account="account" />
     </div>
 
-    <div class="misc">
-      <div class="total-tokens">
-        <span class="digit">
-          {{ totalTokens }}
-        </span>
-        <span class="wording">
-          {{ $t('pages.fungible-tokens.tokens') }}
-        </span>
-      </div>
-    </div>
+    <AccountCardConsensus
+      v-if="isMultisigDashboard"
+      :current-account="account"
+    />
+    <AccountCardTotalTokens
+      v-else
+      :current-account="account"
+      :selected="selected"
+    />
   </BtnBase>
 </template>
 
 <script lang="ts">
-import { computed, defineComponent } from '@vue/composition-api';
-import { IAccount } from '../../types';
-import { useGetter } from '../../composables/vuex';
+import { computed, defineComponent, PropType } from '@vue/composition-api';
 import { getAddressColor } from '../utils/avatar';
+
 import AccountInfo from './AccountInfo.vue';
 import BalanceInfo from './BalanceInfo.vue';
 import BtnBase from './buttons/BtnBase.vue';
+import AccountCardTotalTokens from './AccountCardTotalTokens.vue';
+import AccountCardConsensus from './AccountCardConsensus.vue';
+
+import type { IAccount, IMultisigAccount } from '../../types';
+import { useMultisigAccounts } from '../../composables';
 
 export default defineComponent({
   components: {
+    AccountCardConsensus,
+    AccountCardTotalTokens,
     AccountInfo,
     BalanceInfo,
     BtnBase,
   },
   props: {
-    accountIdx: { type: Number, required: true },
+    account: { type: Object as PropType<IAccount | IMultisigAccount>, required: true },
```

Comment:
In my opinion one prop should not receive two totally different object structures.
It would be total hell when trying to write some `setup` logic that uses this prop data.